lib/postgres_ext/serializers/active_model/array_serializer.rb

```diff
@@ -145,8 +141,25 @@ def _process_has_many_relation(key, association_reflection, relation_query, ids_
       { table: cte_name, ids_column: id_column_name, foreign_key: association_reflection.foreign_key }
     end
 
-    def _visitor
-      @_visitior ||= object.klass.connection.visitor
+    def _to_sql(arel, bind_values = nil)
+      @_visitor ||= object.klass.connection.visitor
+      if @_visitor.method(:accept).arity == 2
+        collector = Arel::Collectors::SQLString.new
+        collector = @_visitor.accept arel, collector
+        res = collector.value
+        unless bind_values.nil?
+          bind_values.each_with_index do |bv, i|
+            if bv[1].is_a? String
+              res = res.gsub(/\$#{i+1}(\b|\Z)/, "'#{bv[1].gsub("'", "''")}'")
+            else
```

Comment:
What case is this line covering? Is it possible to have some part of Arel perform the required replacement? This, to me, feels like it might be something that would be susceptibility to sql injection.